src/App.js

```diff
@@ -1,13 +1,11 @@
 import React from "react";
 import { Route, Routes } from "react-router-dom";
 import Home from "pages/Home";
-import Navbar from "components/Shared/Navbar/index.jsx";
 import "assets/style/app.scss";
 
 function App() {
   return (
-    <div>
-      <Navbar />
+    <div className="app">
       <Routes>
         <Route path="/" element={<Home />} />
```

Comment:
Is there any specific need for that? or else just for practice?

Reply:
for practice. Thank you!